src/common/scheduler/policy/alternate_policy.go

```diff
@@ -0,0 +1,185 @@
+package policy
+
+import "github.com/vmware/harbor/src/common/scheduler/task"
+import "errors"
+import "time"
+
+//AlternatePolicyConfiguration store the related configurations for alternate policy.
+type AlternatePolicyConfiguration struct {
```

Comment:
The reason why implement it like this is to provide flexible compatibilities for future possible more complicated policy. This policy can support 'hourly','daily','weekly' even 'yearly'. It can also supports **ly in a specified scope. As the logic is very similar, doing this can avoid redevelopment. So I insist suggesting to keep this policy as it is.